src/axom/sina/core/Document.cpp

```diff
@@ -15,8 +15,8 @@
 #include "axom/sina/core/Document.hpp"
 
 #include "axom/config.hpp"
-#include "axom/core/Path.hpp"
-#include "axom/core/utilities/StringUtilities.hpp"
+#include "axom/core.hpp"
```

Comment:
@rhornung67 This answers our discussion on Monday about whether Sina requires Core.